website/source/docs/drivers/raw_exec.html.md

```diff
@@ -19,7 +19,8 @@ As such, it should be used with extreme care and is disabled by default.
 The `raw_exec` driver supports the following configuration in the job spec:
 
 * `command` - The command to execute. Must be provided.
-
+* `artifact_source` – Source location of an executable artifact. Must be accessible
```

Comment:
We should add some indication of what "accessible from the Nomad client" means. I suspect this differs slightly from client to client but for any general stuff we can use a shared page like [this one](https://nomadproject.io/docs/jobspec/environment.html).